app/assets/stylesheets/darkswarm/_shop-navigation.css.scss

```diff
@@ -31,72 +33,118 @@
 
     #distributor_title h3 {
       margin-top: 0;
+      padding-top: 0.45em;
+
       @media all and (max-width: 768px) {
           margin-bottom: 8px;
       }
     }
+  }
+}
+
+ordercycle {
+  float: right;
+  background: $secondary-blue;
+  color: $white;
+  width: 100%;
+  border-radius: 0.75em 0.75em 0 0;
+  position: absolute;
+  right: 1em;
+  padding: 1.65em 0 2.5em;
+  margin-top: 1em;
 
+  p {
+    max-width: 400px;
+  }
+
+  h4 i {
+    margin-right: 0.3rem;
+  }
+
+  @media all and (max-width: 1024px) {
+    float: none;
+    padding: 0.5em 1em;
+    width: 100%;
+    margin-top: 0;
+    display: inline-block;
+    border-radius: 0;
+    position: relative;
+    right: 0;
+
+    p {
+      max-width: 100%;
+    }
+  }
+
+  @media all and (max-width: 768px) {
+    padding: 0.5em 1em 0.75em;
+  }
+
+  form.custom {
+    text-align: center;
+  }
+
+  .order-cycle-select {
+    border: 1px solid $light-blue;
+    display: inline-block;
+    font-size: 1em;
+    border-radius: 0.5em;
+
+    .select-label {
+      background-color: $light-blue;
+      display: inline-block;
+      border-radius: 0.5em 0 0 0.5em;
+      float: left;
+      font-size: 1em;
+      line-height: 1.5em;
+      padding: 0.5em 1em;
+      height: 2.35em;
+    }
 
+    select {
+      width: inherit;
+      display: inline-block;
+      color: $white;
+      background-color: transparent;
+      border: 0;
+      margin-bottom: 0;
+      font-size: 1em;
+      line-height: 1.5em;
+      padding: 0.5em 1.25em 0.5em 0.75em;
+      height: 2.35em;
+      background-image: url('data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiIHN0YW5kYWxvbmU9Im5vIj8+CjxzdmcKICAgeG1sbnM6ZGM9Imh0dHA6Ly9wdXJsLm9yZy9kYy9lbGVtZW50cy8xLjEvIgogICB4bWxuczpjYz0iaHR0cDovL2NyZWF0aXZlY29tbW9ucy5vcmcvbnMjIgogICB4bWxuczpyZGY9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkvMDIvMjItcmRmLXN5bnRheC1ucyMiCiAgIHhtbG5zOnN2Zz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciCiAgIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIKICAgaWQ9InN2ZzQiCiAgIHhtbDpzcGFjZT0icHJlc2VydmUiCiAgIGVuYWJsZS1iYWNrZ3JvdW5kPSJuZXcgMCAwIDYgMyIKICAgdmlld0JveD0iMCAwIDYgMyIKICAgaGVpZ2h0PSIzcHgiCiAgIHdpZHRoPSIyNHB4IgogICB5PSIwcHgiCiAgIHg9IjEycHgiCiAgIHZlcnNpb249IjEuMSI+PG1ldGFkYXRhCiAgICAgaWQ9Im1ldGFkYXRhMTAiPjxyZGY6UkRGPjxjYzpXb3JrCiAgICAgICAgIHJkZjphYm91dD0iIj48ZGM6Zm9ybWF0PmltYWdlL3N2Zyt4bWw8L2RjOmZvcm1hdD48ZGM6dHlwZQogICAgICAgICAgIHJkZjpyZXNvdXJjZT0iaHR0cDovL3B1cmwub3JnL2RjL2RjbWl0eXBlL1N0aWxsSW1hZ2UiIC8+PGRjOnRpdGxlPjwvZGM6dGl0bGU+PC9jYzpXb3JrPjwvcmRmOlJERj48L21ldGFkYXRhPjxkZWZzCiAgICAgaWQ9ImRlZnM4IiAvPjxwb2x5Z29uCiAgICAgc3R5bGU9ImZpbGw6I2ZmZmZmZjtmaWxsLW9wYWNpdHk6MSIKICAgICBpZD0icG9seWdvbjIiCiAgICAgcG9pbnRzPSI1Ljk5MiwwIDIuOTkyLDMgLTAuMDA4LDAgIiAvPjwvc3ZnPg==');
```

Comment:
can you please explain what this is?
is this the way we include new images in css?

Reply:
It overrides the default provided by foundation, which is in the same format. This replaces the black caret with a white one.

Reply:
ok, thanks for explaning. This is the only base64 image outside the stripe connect css. I wonder what is the rule for images. I think it would be better to have the image in app/assets/images.

Reply:
Moved :+1:

Reply:
I added some minor display improvements as well.